test/clients.test.js

```diff
@@ -0,0 +1,140 @@
+// testing libraries
+import 'chai';
+import { expect } from 'chai';
+import { fetchMock } from 'fetch-mock';
+
+// testing helpers
+import fs from 'fs';
```

Comment:
Can someone please explain to me why this works, but if I remove the curly braces from the chai imports it blows up in my face?

Reply:
Do you mean doing `import expect from 'chai'` instead of `import { expect } from 'chai'`?

Reply:
Well, just in case - here are some helpful links. Your first import is only getting the side-effects from the module, your second one is destructuring the module exports object coming from chai. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Import_a_module_for_its_side_effects_only

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Import_a_single_export_from_a_module

When you do `import expect from 'chai'`, what's actually happening is you're assigning all of the module exports to `expect`.

```
expect = {foo: 'bar', fizz: 'buzz', expect: () => {}}
```

But if you use the destructuring syntax something like this
```
import {expect, foo} from 'chai';
````
Expands essentially to this
```
expect = chai.module.exports.expect;
foo = chai.module.exports.foo;
```

Reply:
Thanks!